app/models/manageiq/providers/redhat/infra_manager.rb

```diff
@@ -22,6 +22,10 @@ class ManageIQ::Providers::Redhat::InfraManager < ManageIQ::Providers::InfraMana
 
   supports :provisioning
   supports :refresh_new_target
+  supports :vm_import do
+    # The version of the RHV needs to be at least 4.1.5 due to https://bugzilla.redhat.com/1477375
+    unsupported_reason_add(:vm_import, _('RHV version 4.1.5+ required')) unless version_at_least?('4.1.5')
```

Comment:
@smelamud  would you like to maintain the previous message "Cannot import to a RHV provider of version < 4.1.5" ? so both action (vm import) and requirement (RHV > 4.1.5) are included ? 

(not insisting on it)

Reply:
@masayag Yes, it is better message indeed. Done.